### DISCUSSION TITLE
What noisy errors should Sentry filter out next?

### DISCUSSION THREAD

**Discussion by rachrwang** [getsentry/sentry#72365] [Problem Understanding]
Not all errors are created equal - some point to critical problems in code while others are pure noise. We built [Inbound Filters](https://docs.sentry.io/concepts/data-management/filtering/) to automatically discard noisy errors, and hopefully make your Sentry experience more actionable.

Here's the current set of filters that Sentry provides out of the box:
<img width="770" alt="Screenshot 2024-06-07 at 5 22 13 PM" src="https://github.com/getsentry/sentry/assets/82916311/566cc1c8-12cd-4cd0-8909-bd2400366d23">

Despite our best efforts to update this list of filters when we identify new errors, we know new ones crop up all the time and our lists aren't exhaustive. 

**This is where we'd like to ask for your help:** what other errors should Sentry be filtering out? Please be as specific as possible, and link out to examples where possible. 

We'll use the ideas we crowdsource to inform our next sets of updates. Thank you!

**Comment by smeubank** [comment]
We are considering a solution in the JavaScript SDK which could help detect what might be noise from browser extensions, for example, and prevent this noise from appearing in the issues feed.

https://github.com/getsentry/sentry-javascript/issues/10882#issuecomment-2137700863

**Reply by smeubank** [reply]
while this can be done in the JS SDK, we could also consider server side implementations which similar detects events which have these signs of being from extensions and kicking them out

**Comment by collectedmind** [comment]
We tend to get a ton of errors from third party initializations (like ad marketplaces) that end up flooding our logs. Curious if that qualifies, or if that is already handled in a way I haven't found yet.

**Reply by smeubank** [reply]
Yes this counts. So there are some existing ways, but they are not fool proof. The following changelog, covers some existing methods, and a new one which you can use with one of our bundler plugins.

https://sentry.io/changelog/ignore-errors-that-dont-come-from-your-code/

**Reply by n-studio** [reply]
I'm using `allowUrls` and yet I still get errors from `https://www.googletagmanager.com/gtm.js`.

**Comment by danielbeardsley** [comment]
We see a *lot* of errors for chrome extensions that I swear we used to be able to detect in the stack trace, seeing `chrome-extension://`. Something in sentry has presumably mutated the stack traces and now replaces that with `app:///` making it indistinguishable from our own errors.


**Here's a line from a chrome extension:**  (This should be ignored, but the `chrome-extension://` part has been replaced with `app:///` by something in sentry)
```
app:///static/mainWorld/interface.js at line 1:1
```

Here's a line from a piwik pro analytics container (hard to act on, but shouldn't be ignored):
```
app:///ppms.js in m at line 38:301
```

Here's a line from our app code:
```
../../packages/shared/helpers/generic-helpers.ts in timeAsync at line 44:11
```

Here's a line from within next.js platform code: (Not clear why this isn't in `node_modules` or something, but that's likely not sentry's problem)
```
../../../src/client/components/use-reducer-with-devtools.ts in useUnwrapState at line 82:23
```

**Comment by rdevries-capland** [comment]
While this is not necessarily an error to filter, something that I am unable to get filtered out within the health checks is the uptime monitor from Zoraxy.

Here is a sample snippet for reference:

![Screenshot 2025-01-14 073123](https://github.com/user-attachments/assets/811d6118-7890-4aea-8311-c3d2ae3cb9c9)

![image](https://github.com/user-attachments/assets/534d16d1-8642-47e9-ae6d-e6dc9ac14d5a)

This is on Sentry version 24.12.1 self-hosted.

**Comment by TRINNO112** [comment]
https://github.com/user-attachments/assets/322d2d20-1ad1-411e-a025-24fcab0dd2a7

Some sort of weird shaking effect is there when I was trying to actually change the playback speed in the replay menu
